tools/benchmark/benchmark.cpp

```diff
@@ -54,7 +54,9 @@ void Benchmark::logQueryInfo(
 void Benchmark::log(uint32_t runNum, QueryResult& queryResult) const {
     auto querySummary = queryResult.getQuerySummary();
     auto actualOutput = testing::TestHelper::convertResultToString(queryResult);
-    spdlog::info("Run number: {}", runNum);
+    spdlog::info("Run number: {}", // NOLINT(clang-analyzer-optin.cplusplus.UninitializedObject):
```

Comment:
Can we use string_format here as well?

Reply:
Yes, but spdlog has its own formatter, so it doesnt make much sense. In either case we pay the price of a spdlog include.

However, in the future we could swap logging packages if we so desire.